tests/test_operator.py

```diff
@@ -980,6 +980,36 @@ def test_equations_mixed_timedim_stepdim(self):
         assert trees[0][-1].nodes[1].write == u2
 
 
+@skipif_yask
+class TestRegions(object):
+
+    def test_domain_vs_interior(self):
+        """
+        Tests regions work properly in terms of code generation and runtime
+        argument derivation.
+        """
+        grid = Grid(shape=(4, 4, 4))
+        x, y, z = grid.dimensions
+        t = grid.stepping_dim  # noqa
+
+        u = TimeFunction(name='u', grid=grid)  # noqa
+        eqs = [Eq(u.forward, u + 1),
+               Eq(u.forward, u.forward + 2, region=INTERIOR)]
+
+        op = Operator(eqs, dse='noop', dle='noop')
+        trees = retrieve_iteration_tree(op)
+        assert len(trees) == 2
+
+        op.apply()
```

Comment:
SO how many time steps does it run, only updates t=1 so one?

Reply:
it's exactly the same behaviour as ``master`` ATM.

What I wrote on Slack was for the next PR